src/main/java/io/openliberty/tools/intellij/actions/RunLibertyDevTask.java

```diff
@@ -51,28 +52,41 @@ public void update(@NotNull AnActionEvent e) {
             //if the action event data context returns a null project, just return and let successive update calls modify the presentation
             return;
         }
+        handleLibertyTreeEvent(e, project, true);
+    }
+
+    private static void handleLibertyTreeEvent(@NotNull AnActionEvent e, Project project, boolean isUpdate) {
```

Comment:
Is there a reason this method was made `static`? It does not seem like it needs to be.

Reply:
No, it was unnecessary. I've already implemented the modification.